src/transforms/transform-secret-scanning-alert.ts

```diff
@@ -0,0 +1,57 @@
+import {
+	JiraVulnerabilityBulkSubmitData, JiraVulnerabilitySeverityEnum, JiraVulnerabilityStatusEnum
+} from "interfaces/jira";
+import { getGitHubClientConfigFromAppId } from "utils/get-github-client-config";
+import { WebhookContext } from "routes/github/webhook/webhook-context";
+import { transformRepositoryId } from "~/src/transforms/transform-repository-id";
+import { SecretScanningAlertEvent } from "../github/secret-scanning-alert";
+import Logger from "bunyan";
+
+export const transformSecretScanningAlert = async (context: WebhookContext<SecretScanningAlertEvent>, jiraHost: string): Promise<JiraVulnerabilityBulkSubmitData> => {
+	const { alert, repository } = context.payload;
+
+	const githubClientConfig = await getGitHubClientConfigFromAppId(context.gitHubAppConfig?.gitHubAppId, jiraHost);
+	return {
+		vulnerabilities: [{
+			schemaVersion: "1.0",
+			id: `d-${transformRepositoryId(repository.id, githubClientConfig.baseUrl)}-${alert.number}`,
+			updateSequenceNumber: Date.now(),
+			containerId: transformRepositoryId(repository.id, githubClientConfig.baseUrl),
+			displayName: alert.secret_type_display_name || `${alert.secret_type} secret exposed`,
+			description:  "Secret scanning alert",
+			url: alert.html_url,
+			type: "sca",
+			introducedDate: alert.created_at,
+			lastUpdated: alert.updated_at || alert.created_at,
+			severity: {
+				level: JiraVulnerabilitySeverityEnum.CRITICAL
+			},
+			identifiers: [],
+			status: transformGitHubStateToJiraStatus(context.action, context.log),
```

Comment:
Won't we fetch the `state` here? Should use that instead and then the `context.action` as a fallback?